astropy/io/fits/scripts/fitscheck.py

```diff
@@ -136,17 +137,17 @@ def verify_checksums(filename):
     except UserWarning as w:
         remainder = '.. ' + ' '.join(str(w).split(' ')[1:]).strip()
         # if "Checksum" in str(w) or "Datasum" in str(w):
-        log.warn('BAD %r %s' % (filename, remainder))
+        warnings.warn('BAD %r %s' % (filename, remainder), AstropyWarning)
```

Comment:
As I wrote in the original issue, use of `warnings.warn` in this script are not actually appropriate.